### DISCUSSION TITLE
How to use maxDate can anyone give an example?

### DISCUSSION THREAD

**Discussion by 19PA1A0548** [wojtekmaj/react-calendar#601] [Q&A] [answered]
I want to set the user to pick only dates before a particular date and I should use maxDate can anyone give me an example of how to use it.

**Comment by wojtekmaj** [comment, marked as answer]
As documented in README, `maxDate` accepts [Date() object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date):

https://codesandbox.io/s/react-calendar-max-date-ytqfly?file=/src/App.js
